### DISCUSSION TITLE
Issues upgrading from 2.3.130 to 2.3.140

### DISCUSSION THREAD

**Discussion by logz4u** [Security-Onion-Solutions/securityonion#8385] [Unsupported Versions]
I know that this issue has been posted at least twice, but neither of those threads provided a working solution for my problem, so I am going to start a new thread. 

My current setup is an air-gapped, standalone, ManagerSearch Node. It is only used for testing (so we don't mess with our production nodes until we are good to go).  

I downloaded the hotfix'd .140 ISO. 

My situation is exactly the same as [#8324](https://github.com/Security-Onion-Solutions/securityonion/discussions/8324).
After soup is performed, I get one unhandled error:
 `Rendering SLS 'base:filebeat' failed: Jinja variable the template 'filebeat/map.jinja' (imported on line 22) does not export the requested name 'SO'`

Thanks to snapshots, I have been able to do a number of troubleshooting steps such as:
1. Using @weslambert suggestion of 
`yum history packages-list salt-master # (to find the 3004.2 salt update)
 yum history undo # (update number, to the left of the output )
 sudo BRANCH=hotfix/2.3.140 soup`

2. With fresh pre soup snapshot, I deleted the additional code pertaining to filebeat modules (netflow and palo alto) in my `local/pillar/minions/standalone.sls` file.  I then performed soup and got the same error. 

I am puzzled and will take any wisdom or advice =). Also sorry for my formatting...new to github.

**Comment by weslambert** [comment]
Please check to ensure there is not a local copy of `init.sls` in `/opt/so/saltstack/local/salt/filebeat/`. If there is, move it out of the directory and migrate your changes to a copy of the new `init.sls` from `/opt/so/salstack/default/salt/filebeat/`. 

If you don't need to make changes to the defaults, do not put a copy of the file in the local directory, as it could cause issues in future upgrades.
